pkg/aws/ocpgwdeployer.go

```diff
@@ -51,6 +53,9 @@ func NewOcpGatewayDeployer(cloud api.Cloud, msDeployer ocp.MachineSetDeployer, i
 		return nil, errors.New("the cloud must be AWS")
 	}
 
+	aws.nodeSGSuffix = "-worker-sg"
+	aws.controlPlaneSGSuffix = "-master-sg"
```

Comment:
It makes sense to set default values also under setSubnetSuffixes function

Reply:
Done.